lib/components/viewers/trip-viewer.js

```diff
@@ -88,6 +88,13 @@ class TripViewer extends Component {
       timeZone: homeTimezone
     })
 
+    const fromIndex = tripData?.stops?.findIndex(
+      (stop) => stop.stopId === viewedTrip?.fromStopId
+    )
+    const toIndex = tripData?.stops?.findIndex(
+      (stop) => stop.stopId === viewedTrip?.toStopId
+    )
+
```

Comment:
Could pull this out into its own function like `findStationIndex` and then pass the `fromStopId` or `toStopId` respectively? Up to you!

Reply:
I think that would wind up being more code overall... Especially since this view is on its way out anyway I think this can be a future improvement